File: src/main/java/ir/reservs/reservs/ui/main/password/PasswordContract.java
package ir.reservs.reservs.ui.main.password;

import ir.reservs.reservs.ui.base.BaseFragmentContract;

public class PasswordContract {
  interface View extends BaseFragmentContract.View {
    void passwordChangedSuccessful();
  }

  interface Presenter extends BaseFragmentContract.Presenter<View> {
    void changePassword(String current_password, String new_password);
  }
}

File: src/main/java/ir/reservs/reservs/ui/main/password/PasswordContract.kt
package ir.reservs.reservs.ui.main.password;

import ir.reservs.reservs.ui.base.BaseFragmentContract;

class PasswordContract {
    interface View : BaseFragmentContract.View {
        fun passwordChangedSuccessful()
    }

    interface Presenter : BaseFragmentContract.Presenter<View> {
        fun changePassword(current_password: String, new_password: String)
    }
}